docs/index.md

```diff
@@ -21,5 +21,6 @@ You may want to jump to:
 - [GitHub repo for `jj`](https://github.com/martinvonz/jj)
 - Overview of `jj` in the repo's [README](https://github.com/martinvonz/jj?tab=readme-ov-file#readme)
 - [Installation and Setup](install-and-setup.md)
+- [Roadmap](roadmap.md)
```

Comment:
I'd put it at the end of the list and call it "Development Roadmap".

You could also add it to `mkdocs.yml`. I'm not sure which section it best fits into, but perhaps "Technical details"? It's not very prominent, but linking to it from README and the homepage is probably prominent enough. Or perhaps as the first entry in "Design Docs"?

I also thought of putting it in "Contributing", but I am not sure I like that idea. Putting it there might make people feel like they are supposed to read and follow the roadmap before contributing.

https://github.com/martinvonz/jj/blob/250a28b5a581733d0ea5742d5dbccbd22eddc4ad/mkdocs.yml#L126

Reply:
I care more about clarifying what kind of roadmap this is than about putting it at the end of the list. You could also keep it where it is, or put it in a new paragraph after this list.

Reply:
I changed it to "Development Roadmap" and moved it last. I don't want to put it in "Contributing" because I think it can also be of interest to people who are not interested in contributing.